client/src/SubscriptionSelection.css

```diff
@@ -0,0 +1,70 @@
+@font-face {
+  font-family: "Inter-Thin";
+  src: url("Inter-Thin.ttf");
+}
+@font-face {
+  font-family: "Inter-Bold";
+  src: url("Inter-Bold.ttf");
+}
+.subscriptionSelectionBackground {
+  background-image: url("./goldenflower.jpg");
+  background-position: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  width: 100vw;
+}
+.pageContents {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  position: relative;
+}
+.header {
+  font-family: "Inter-Thin";
+  font-size: 73px;
+  font-style: italic;
+  font-weight: 400;
+}
+.cards {
+  align-items: center;
+  background-color: white;
+  bottom: 0;
+  display: flex;
+  height: 80vh;
+  justify-content: center;
+  margin: auto;
+  width: 80vw;
+}
+
+.button {
+  /* transition-duration: 0.4s; */
```

Comment:
Can this comment be removed?

Reply:
We were thinking to keep this as we would like to implement a transition on the icon as well although we have not yet found how to include this css property in in line styling. We were considering creating a CSS file just for the Icon - but maybe that is excessive. What do you think?